stdlib/dynarray.ml

```diff
@@ -267,6 +275,41 @@ end = struct
         done;
       end
 
+    let blit src src_dummy src_pos dst dst_dummy dst_pos ~len =
```

Comment:
For this kind of critical functions with several arguments of the same type, I like to use labels, not for convenience but for safety: they prevent getting the order wrong at the call site, e.g., swapping source and destination.